odoo/tools/date_utils.py

```diff
@@ -55,7 +55,11 @@ def get_fiscal_year(date, day=31, month=12):
     :return: A tuple (date_from, date_to) having the same object type as the 'date' parameter.
     '''
     max_day = calendar.monthrange(date.year, month)[1]
-    date_to = type(date)(date.year, month, min(day, max_day))
+    # Force at 28 February instead of 29 in case of no leap year.
+    if month == 2 and day == 29 and max_day == 28:
+        date_to = type(date)(date.year, 2, 28)
+    else:
+        date_to = type(date)(date.year, month, min(day, max_day))
```

Comment:
Shorter:
```suggestion
        if month == 2 and day in (28, max_day):
            date_to = type(date)(date.year, 2, max_day)
```
(the condition below is no longer necessary)

Reply:
> Shorter:
> 
> (the condition below is no longer necessary)

Hello @smetl 

We need the `else` condition because the `if` condition is only for `February` month so if the month is not `February` then we need to find `date_to` for those months also.

Do let me know your suggestion on it!

Thank You!

Reply:
@uso-odoo 
```suggestion
        if (month == 2 and day in (28, max_day)) or (month != 2 and day in (30, max_day)):
            day = max_day
        date_to = type(date)(date.year, month, min(day, max_day))
```

Reply:
> @uso-odoo

Hello @smetl 

The changes have been made! Can you please look into it?

Thank You!